xhype/xhype/cdef2rust.py

```diff
@@ -0,0 +1,27 @@
+import sys
```

Comment:
what license is this? where is it from? Do you need it here or can people get it some other way.

Reply:
Both cstruct2rust.py and cdef2rust.py are written by myself and are used for converting some C code to Rust. It is not necessary for the program. I just put here for future development. Struct in bios.rs and linux.rs and constants in msr.rs are generated by the two scripts.